update readme (#127)
Qiskit/documentation: README.md

# Qiskit docs

The documentation content home for https://docs.quantum-computing.ibm.com. Note this repo does not contain content for https://learning.quantum-computing.ibm.com/ or https://qiskit.org. Refer to https://github.com/Qiskit/qiskit to make changes to the docs at https://qiskit.org/documentation.

# Improving IBM Quantum & Qiskit Documentation

> Maintaining up to date documentation is a huge challenge for any software project, especially in a field like quantum computing where the pace at which new research and technological capabilities get released is incredibly fast. As a result, we greatly appreciate any who take the time to support us in keeping this content accurate and up to the highest quality standard possible to benefit the broadest range of users.

Reviewer: ```suggestion
Maintaining up-to-date documentation is a huge challenge for any software project, especially in a field like quantum computing where the pace at which advances in new research and technological capabilities happen incredibly fast. As a result, we greatly appreciate any who take the time to support us in keeping this content accurate and up to the highest quality standard possible to benefit the broadest range of users.
```

Read on for more information about how to support this project:

## Best ways to contribute to Documentation
### 1. Report bugs, inaccuracies or general content issues

> This is the quickest, easiest and most helpful way to contribute to this project and improve the quality of Qiskit and IBM Quantum documentation. There are a few different ways to report issues depending on where it was found:

Reviewer: ```suggestion
This is the quickest, easiest, and most helpful way to contribute to this project and improve the quality of Qiskit and IBM Quantum documentation. There are a few different ways to report issues, depending on where it was found:
```

> - For problems you've found in the [Qiskit API Reference](https://docs.quantum-computing.ibm.com/api/qiskit) section, open an issue in the Qiskit repo [here](https://github.com/Qiskit/qiskit/issues/new/choose)

Reviewer: ```suggestion
- For problems you've found in the [Qiskit API Reference](https://docs.quantum-computing.ibm.com/api/qiskit) section, open an issue in the Qiskit repo [here](https://github.com/Qiskit/qiskit/issues/new/choose).
```

> - For problems you've found in the [Qiskit Runtime IBM Client](https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/runtime_service) section, open an issue in the Qiskit IBM Runtime repo [here](https://github.com/Qiskit/qiskit-ibm-runtime/issues/new/choose)

Reviewer: ```suggestion
- For problems you've found in the [Qiskit Runtime IBM Client](https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/runtime_service) section, open an issue in the Qiskit IBM Runtime repo [here](https://github.com/Qiskit/qiskit-ibm-runtime/issues/new/choose).
```

- For problems you've found in any other section of [docs](https://docs.quantum-computing.ibm.com), open a content bug issue [here](https://github.com/Qiskit/documentation/issues/new/choose).

### 2. Suggest new content

If you think there are gaps in our documentation, or sections that could be expanded upon, we invite you to open a new content request issue [here](https://github.com/Qiskit/documentation/issues/new/choose). 

> Not every new content suggestion is a good fit for docs, nor are we able to prioritise every request immediately. However, we will do our best to respond to content requests in a timely manner, and we greatly appreicate our community's efforts in generating new ideas.

Reviewer: ```suggestion
Not every new content suggestion is a good fit for docs, nor are we able to prioritize every request immediately. However, we will do our best to respond to content requests in a timely manner, and we greatly appreciate our community's efforts in generating new ideas.
```

Author: 😑😑😑😑

> If you are interested in writing the new content yourself, or already have some draft work you think could be integrated, please also mention that in the issue description. If your content suggestion is accepted we will let you know and work with you to get the content written and reviewed.

Reviewer: ```suggestion
If you are interested in writing the new content yourself, or already have some draft work you think could be integrated, please also mention that in the issue description. If your content suggestion is accepted, we will let you know and work with you to get the content written and reviewed.
```

> Please note: we DO NOT accept unsolicited PRs for new pages or large updates to existing content. The content that we include in docs is carefully planned and curated by our content team and must go through the appropriate review process to ensure the quality is of the highest possible standard before deploying to production. As a result we are very selective with which content suggestions are approved and it is unlikely that PRs submitted without an associated approved content request will be accepted.

Reviewer: ```suggestion
Please note: we DO NOT accept unsolicited PRs for new pages or large updates to existing content. The content that we include in docs is carefully planned and curated by our content team and must go through the appropriate review process to ensure the quality is of the highest possible standard before deploying to production. As a result we are very selective with which content suggestions are approved, and it is unlikely that PRs submitted without an associated approved content request will be accepted.
```

> ### 3. Validate exisiting issues

Reviewer: ```suggestion
### 3. Validate existing issues
```

Author: damn you can really tell I wrote this page in a rush can't you lol

> You can help the maintainer team prioritise already open issues by doing the following:

Reviewer: ```suggestion
You can help the team prioritize already-open issues by doing the following:
```

Author: Is there a reason you don't like the term "maintainer"? I'm not strongly attached to it, just curious

> - For bug reports, leave a comment in the issue if you have also been experiencing the same problem and can reproduce it (include as much information as you can, e.g.  browser type, qiskit version etc.)

Reviewer: ```suggestion
- For bug reports, leave a comment in the issue if you have also been experiencing the same problem and can reproduce it (include as much information as you can, e.g.,  browser type, Qiskit version, etc.).
```

- For new content requests, leave a comment or upvote (👍) in the issue if you also would like to see that new content added.

### 4. Fix an open issue

You can look through the open issues we have in this repo and address them with a PR. We recommend focusing on issues with the "good first issue" or "help wanted" label. 

> Before getting started on an issue remember to do the following:

Reviewer: ```suggestion
Before getting started on an issue, remember to do the following:
```

> 1. Read the Code of Conduct (TODO add link)

Reviewer 2: Should we link to https://github.com/Qiskit/qiskit/blob/main/CODE_OF_CONDUCT.md for now?

> 2. Check for issues with the "good first issue" or "help wanted" label

Reviewer 2: We don't have these labels. If we want them, maybe consolidate around one?

Reviewer: I added a "good first issue" label just now.

Reviewer 2: Thanks! I personally like "good first issue" more than "help wanted". I think it makes sense to have only one. They serve such a similar purpose.

Author: we've used both pretty effectively in the [qiskit repo](https://github.com/Qiskit/qiskit/blob/main/CONTRIBUTING.md#choose-an-issue-to-work-on) in the past, but happy to keep it just to GFI for now and we can update later if we need

3. Select an issue that is not already assigned to someone and leave a comment to request to be assigned

> Once you have an issue to work on see the "How to work with this repo" section below to get going, then open a PR.

Reviewer: ```suggestion
Once you have an issue to work on, see the "How to work with this repo" section below to get going, then open a PR.
```

> Before opening a PR remember to do the following:

Reviewer: ```suggestion
Before opening a PR, remember to do the following:
```

1. Check that you have addressed all the requirements from the original issue
2. Run the spell checker
3. Use the GitHub "fixes" notation to [link your PR to the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) you are addressing

# How to work with this repo

> ## Pre-requisites to building the docs locally

Reviewer: ```suggestion
## Prerequisites to building the docs locally
```
